database/pom.xml

```diff
@@ -50,21 +48,11 @@
         <artifactId>h2</artifactId>
         <version>${version.h2}</version>
       </dependency>
-      <dependency>
-        <groupId>com.oracle.jdbc</groupId>
-        <artifactId>ojdbc6</artifactId>
-        <version>${version.oracle}</version>
-      </dependency>
       <dependency>
         <groupId>com.oracle.jdbc</groupId>
         <artifactId>ojdbc7</artifactId>
         <version>${version.oracle}</version>
       </dependency>
```

Comment:
We can consider dropping this one and using only Orcl 19 driver if the effort is small. Compatibility should be in place:
https://docs.oracle.com/en/database/oracle/oracle-database/19/jjdbc/JDBC-getting-started.html#GUID-926E5324-D89A-4A00-B1AE-975C1089F0EA

Reply:
After checking which exact Oracle 12 we use in CI, I think it is better to keep it as it is. We use Oracle 12.1.0.2 which is only compatible with ojdbc7 or ojdbc6. 
![image](https://user-images.githubusercontent.com/3015690/190400216-519a8e3d-5c24-42bc-bb85-d46e28ccd36e.png)